NOTICE

```diff
@@ -0,0 +1,42 @@
+# Notices for Eclipse Mylyn
+
+This content is produced and maintained by the Eclipse Mylyn project.
+
+* Project home: https://projects.eclipse.org/projects/tools.mylyn
+
+## Trademarks
+
+Eclipse Mylyn is a trademark of the Eclipse Foundation.
+
+## Copyright
+
+All content is the property of the respective authors or their employers. For
+more information regarding authorship of content, please consult the listed
+source code repository logs.
+
+## Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License v. 2.0 which is available at
+http://www.eclipse.org/legal/epl-2.0.
+
+SPDX-License-Identifier: EPL-2.0
+
+## Source Code
+
+The project maintains the following source code repositories:
+
+* https://github.com/eclipse-mylyn/org.eclipse.mylyn.git
+* https://github.com/eclipse-mylyn/org.eclipse.mylyn.all.git
+* https://github.com/eclipse-mylyn/org.eclipse.mylyn.builds.git
+* https://github.com/eclipse-mylyn/org.eclipse.mylyn.commons.git
+* https://github.com/eclipse-mylyn/org.eclipse.mylyn.context.git
+* https://github.com/eclipse-mylyn/org.eclipse.mylyn.context.mft.git
```

Comment:
I think that repository no longer active.
It is not included in the Mylyn.setup.

Reply:
Done